### ISSUE TITLE
Can't set context register "Show PAC",it would cause error

### ISSUE THREAD

**Issue by fjh1997** [NationalSecurityAgency/ghidra#2671]
**Describe the bug**
I need to set "Show PAC" flag to 1 in order to decompile pac related instruction Supported in ARMv8.3.
as shown in #2488 

however I can't set context register "Show PAC",it would cause error:

> Context register change conflicts with one or more instructions

**To Reproduce**
1. import binary file with pac instruction
2. right click in the listing panel，click “set register value”
![image](https://user-images.githubusercontent.com/25281816/105279974-b8bc8b00-5be3-11eb-851b-17296202778d.png)

3. set context register "Show PAC" to 1

![image](https://user-images.githubusercontent.com/25281816/105280093-f7eadc00-5be3-11eb-9a82-184eca9f8464.png)

![image](https://user-images.githubusercontent.com/25281816/105280139-0c2ed900-5be4-11eb-93a8-3f179021b042.png)
**Expected behavior**
context register value changed, and I can redecompile to get pac related instruction correctly decompiled

**Attachments**
binary file
[chall.zip](https://github.com/NationalSecurityAgency/ghidra/files/5846864/chall.zip)

**Environment (please complete the following information):**
 - OS: [e.g. macOS 11.1]
 - Java Version: [e.g. 14.0.2]
 - Ghidra Version: [e.g. 9.2.1]

**Comment by LukeSerne** [comment]
I'm reproducing this in Ghidra 12.0.4. Even when setting the register value before auto analysis has run (so nothing has been disassembled yet), I still get this error. I ended up working around this by changing the `.pspec` file as discussed in the linked issue.

This is extra annoying, since the default value for `Show PAC` hides these instructions completely from the decompiler, even though the instruction has a clear influence on the register value, leading to misleading decompiler output. While I understand the appeal of less complex decompiled code, it would be nice if there was a simple way to turn on the more detailed PAC instructions.

**Comment by GhidorahRex** [comment]
You should be able to clear the bytes, change the register, and then re-disassemble, but I understand why that wouldn't be ideal, especially if it's a large binary. The best fix is updating the .pspec file, as you've done.

**Comment by LukeSerne** [comment]
I tried that as well, but that didn't work either 🤔. It would be great if there was a simple toggle to toggle these pcode ops on or off, similar to the unreachable code and read-only toggles for the decompiler output.

**Comment by GhidorahRex** [comment]
> I tried that as well, but that didn't work either 🤔. It would be great if there was a simple toggle to toggle these pcode ops on or off, similar to the unreachable code and read-only toggles for the decompiler output.

I agree, it would be nice if there was a toggle for the pcodeops (in general).

I just tried it and it worked. If you clear and disassemble the bytes of the instructions that use the `*__PACpart` subconstructors and set the context variable then disassemble again the pcodeop will show up in the decompiler. You need to make sure when you right-click and change the register value that you do so over the relevant instructions or bytes. If the instruction is cleared, then you need to make sure you're right-clicking on the first byte of the instruction word to be disassembled.

**Comment by LukeSerne** [comment]
Ah, I see - I selected everything in the listing using ctrl+A, then cleared all of that from the right-click context menu, and then right-clicked again to set the register value. I just clicked at an arbitrary point, thinking it wouldn't matter. Thanks for the additional instructions.

Hopefully this issue can serve as a justification for working on a system to toggle pcodeops. Perhaps the NAN pcodeop handing (which is now configured in the decompiler analysis options) could also be moved into that system.
